lib/common.sh

```diff
@@ -837,6 +839,11 @@ common::sendmail () {
   local cc="$5"
   local file="$6"
 
+  if [[ "$HOSTNAME" =~ google.com ]]; then
```

Comment:
this check is pretty late in the process, no?

also, it seems somewhat redundant with the check in `lib/common.sh`.